submissions/expero1/html-css-popup/style.css

```diff
@@ -0,0 +1,405 @@
+.user-menu a:focus  ~ .popup-menu-wrapper > .popup-menu
+{
+	display: none;
+}
+.popup-menu-wrapper {
+	position: relative;
+	grid-area:popup-menu-wrapper;
+	z-index: 10;
+}
+
+.popup-menu-inner {
+	background-color: white;
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	justify-items: center;
+	padding:5px;
+	gap: 20px 10px;
+}
+
+.popup-menu-inner.hidden {
+	display: none;
+}
+
+/* Product btn*/
+
+.product-btn {
+	display: inline-block;
+	list-style: none;
+	width: 6rem;
+	height: 6rem;
+	padding: 5px;
+/*	border: 1px solid black; /*DEBUG*/
+
+}
+
+.product-btn:hover,
+.product-btn:focus {
+	outline: 2px solid grey;
+	box-shadow: 0 0 15px grey;
+}
+.popup-menu-inner li {
+	list-style: none;
+}
+.popup-menu-inner li a{
+	display: flex;
+	height: 100%;
+	flex-direction: column;
+	justify-content: flex-end;
+	gap: 5px;
+	align-items: center;
+	background-color: white;
+	margin: 0 auto;
+	position: relative;
+}
+
+.product-btn img {
+	display: block;
+/*	border: 1px solid black; /* DEBUG */
+	object-fit:contain;
+	width: 3em;
+	height: 3em;
+	margin: 0 auto;
+}
+
+.product-btn span {
+	text-align: center;
+	display: block;
+	margin-top: 5px;
+}
+
+/* SHOW MORE button*/
+
+.show-more-checkbox {
+	-webkit-appearance:none;
+	-moz-appearance: none;
+	appearance: none;
+	outline: none;
+	display: inline;
+}
+.btn {
+	width: 100%;
+border:none;
+	height: 3rem;
+	padding: 1rem 0;
+	background-color: whitesmoke;
+	text-align: center;
+	text-transform: uppercase;
+	cursor:pointer;
+}
+.btn:hover, .btn:focus{
+background-color: gray;
+}
+.btn:active {
+background-color:dimgray;
+}
+.show-more-btn {
+	display: block;
+	text-transform: uppercase;
+}
+.show-more-checkbox:checked,
+.show-more-checkbox:checked ~ .show-more-btn {
+	display: none;
+}
+
+.show-more-checkbox:checked ~ .hor-line {
+	display: block;
+position:relative;
+/*top:-8px;*/
+	margin: 10px 0 5px 0;
+	height: 1px;
+	border-bottom: 1px solid gray;
+	box-shadow: 0 0 10px gray;
+}
+.show-more-checkbox:checked ~ .popup-menu-inner.hidden {
+	display: grid;
+}
+.show-more-checkbox:focus ~ .show-more-btn {
+background-color: gray;
+}
+
+.show-more-checkbox:checked ~ .even-more-btn {
+	display: block;
+}
+
+.even-more-btn {
+	width: 100%;
+	display: none;
+margin-top: 10px;
+
+	border: none;
+	cursor: pointer;
+	height: 3em;
+}
+.even-more-btn:hover,
+.even-more-btn:focus {
+	background-color: gray;
+}
+
+.notification-menu {
+	grid-area: notification-menu;
+}
+.user-menu{
+	grid-area:user-menu;
+}
+.user-menu img,
+.notification-menu img {
+	object-fit: cover;
+	width: 30px;
+}
+
+main {
+	height: 110vh;
+	background-color: Gainsboro;
+	z-index: -1;
+	position: relative;
+}
+main::after {
+	content: 'main section';
+	color:white;
+	position: absolute;
+	margin: 0 auto;
+	top: 50vh;
+	right: 50%;
+	text-transform: uppercase;
+	z-index: -1;
+	display: block;
+	text-align: center;
+	transform: translateX(50%);
+	font-size: 50px;
+}
+
+footer {
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

Reply:
Thank you. I have read the article with great attention. Corrected according to new knowledge.